modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionCommandListener.java

```diff
@@ -163,4 +308,26 @@ else if (clo.command() instanceof UpsertCommand) {
             return hash;
         }
     }
+
+    /**
+     * @param row Row.
```

Comment:
Please add describing javadoc, what contract for comparing is.

Reply:
Rewrote javadoc.